_careers/4-internships.md

```diff
@@ -51,10 +51,18 @@ We believe in hiring good people and treating them well. This means we look at m
 
 * Have strong, well thought-out values - We love it when you have a vision for a "perfect" Singapore. The ideas and insights you bring will only help our work to become more relevant, more effective, and more appropriate.
 
+### **Internship positions currently open**
+
+* Software Engineer interns (off-cycle, i.e. can intern beyond May-Jul 2021)
```

Comment:
The timeframe here is somewhat ambiguous. Can we perhaps phrase it as the start date from which they can be considered, ie "from August 2021" or similar?

Reply:
@LoneRifle see if changes make sense to you? @cakesoccer fyi